### ISSUE TITLE
[kubeflow v1.3.0-rc.1] cache server certificate secret not being created

### ISSUE THREAD

**Issue by raffaelespazzoli** [kubeflow/pipelines#5507]
I'm trying installing kubeflow v1.3.0-rc.1 with the official manifests and it seems to me that the manifests for creating the certs for the cache-server are not present.
It is still hard to say for certain for sure as the kustomize composition is still very hard to follow. That said, thanks for having removed kfctl and kfdef.

**Comment by Bobgy** [comment]
Also transferred to KFP repo

**Comment by raffaelespazzoli** [comment]
@Bobgy yeah, I intended to open this on the manifests repo. sorry for this., can you add the link to where you transferred it to?
I'm seeing this issue with both minkube 1.17.1 and openshift 4.7 (corresponding to kube 1.20). In both cases I'm using the https://github.com/kubeflow/manifests/apps/pipeline/upstream/env/platform-agnostic-multi-user/?ref=v1.3.0-rc.1 manifests.

When I manually added the cert, I started getting this error from the cache-server log:
```
2021/04/20 12:02:00 open /etc/webhook/certs/cert.pem: no such file or directory
```
it should try to open `tls.crt` not `cert.pem`...

**Comment by Bobgy** [comment]
@raffaelespazzoli the issue seems to be KFP specific, so I suggest putting it here. You can ping corresponding distribution owners here too.

**Comment by Bobgy** [comment]
cache-deployer deployment is responsible for creating the cert, can you report what logs are there?
